app/src/main/java/org/ole/planet/myplanet/ui/team/MyTeamsDetailFragment.kt

```diff
@@ -233,8 +235,8 @@ class MyTeamsDetailFragment : BaseNewsFragment() {
         llRv.visibility = View.GONE
         tab.setText(s)
         listContent.adapter = object : ArrayAdapter<RealmUserModel?>(requireActivity(), android.R.layout.simple_list_item_1, data) {
-            override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
```

Comment:
sorry for being picky, but my android studio actually recommends that naming for convert_View should stay as convertView
<img width="528" alt="Screenshot 2024-07-01 at 3 12 53 PM" src="https://github.com/open-learning-exchange/myplanet/assets/59347000/5386839d-56a3-4c76-bd47-ac95824e2393">

Reply:
The underscore is to avoid overshadowing var convertView on line 239. Given that convertView is used quite a bit, adding the underscore would keep the changes as granular as possible and resolve the warnings.

Reply:
I see what you are saying. I agree that these variables should not have the same name. I'd suggest changing one of their names to be something other than convertView so that you can avoid this warning from Android Studio.

Reply:
I agree with @ewhang5 the change you did does not reduce the warnings use a different variable in camelCase